src/libNetwork/P2PComm.h

```diff
@@ -76,20 +76,23 @@ class P2PComm
     Peer m_selfPeer;
 #endif // STAT_TEST
 
-    ThreadPool m_SendPool{MAXMESSAGE, "SendPool"};
-    ThreadPool m_RecvPool{MAXMESSAGE, "RecvPool"};
+    ThreadPool m_SendPool{MAXMESSAGE / 2, "SendPool"};
```

Comment:
Any rationale to divide by 2? Or maybe can we just edit the constant file

Reply:
I did that. Maybe it should be reverted to the constant file. I did this because always the number of threads were twice of maxmessages which made it easier to know how many threads are being formed

Reply:
I did that. Maybe it should be reverted to the constant file. I did this because always the number of threads were twice of maxmessages which made it easier to know how many threads are being formed

Reply:
OK, I will change constant_local.xml instead of modifying P2PComm.h